compiler/qsc_frontend/src/lower.rs

```diff
@@ -144,7 +144,10 @@ impl With<'_> {
         let exported_hir_ids = exports
             .iter()
             .filter_map(|(res, alias)| match res {
-                resolve::Res::ExportedItem(id) | resolve::Res::Item(id, _) => Some((*id, alias)),
+                resolve::Res::ExportedItem(id, hir_alias) => {
+                    Some((*id, hir_alias.clone().or(alias.clone())))
```

Comment:
I haven't tried this locally, but I'm wondering if this could be adjusted to avoid the appearance of a double clone with something like `hir_alias.or(alias).clone()`